Cython/Utility/ObjectHandling.c

```diff
@@ -2322,7 +2322,10 @@ static CYTHON_INLINE PyObject* __Pyx_PyObject_CallNoArg(PyObject *func) {
         return __Pyx_PyFunction_FastCall(func, NULL, 0);
     }
 #endif
-#ifdef __Pyx_CyFunction_USED
+#ifdef __Pyx_CyFunction_USED && defined(NDEBUG)
```

Comment:
```suggestion
#if defined(__Pyx_CyFunction_USED) && defined(NDEBUG)
```

Reply:
Thanks - not sure how I missed that or why it didn't break

Reply:
I think gcc just ignored everything after the first term (and printed a warning)